src/api/archive_unstable_storageDiff.md

```diff
@@ -0,0 +1,98 @@
+# archive_unstable_storageDiff
+
+**Parameters**:
+
+- `hash`: String containing a hexadecimal-encoded hash of the header of the block whose storage difference will be retrieved.
+
+- `previousHash` (optional): String containing a hexadecimal-encoded hash of the header of the block from which the storage difference will be calculated. The `previousHash` must be an ancestor of the provided `hash`.  When this parameter is provided, the storage difference is calculated between the `hash` block and the `previousHash` block. If this parameter is not provided, the storage difference is calculated between the `hash` block and the parent of the `hash` block.
+
+- `items` (optional): Array of objects. The structure of these objects is found below.
+
+    ```json
+    "items": [
+        {
+            "prefixes": [
+                {
+                    "key": "0x...",
+                    "type": "value" | "hash" | "none",
```

Comment:
I find `"none"` hard to understand and AFAIU it's a way to get all storage diffs and ignore the type of the storage diff.

I suggest rename it to "all" or something like that.

Can you document the "type" options a bit more through? I didn't see any explanation of "none" except in the PR description.